smoke-test/urls.test.js

```diff
@@ -0,0 +1,52 @@
+const {readFileSync} = require('fs');
+const testCases = [];
+
+// Load a suite of smoke tests from a simple text file. File is of the format:
+// [path], [comma separated text assertions]
+const text = readFileSync(require.resolve('./urls.txt'), 'utf8').trim();
+for (const line of text.split(/\r?\n/)) {
+  if (line.length === 0) continue; // allow for blank lines.
+  if (line.match(/^#/)) continue; // allow for comments.
+  const testCase = {assertions: []};
+  line.split(/,/).forEach((token, i) => {
+    if (i === 0) {
+      // The first token is the path that we wish to traverse the headless
+      // browser to:
+      testCase.path = token;
+    } else {
+      // The remaining tokens represent blocks of text in the rendered page
+      // we should check for:
+      testCase.assertions.push(token.trimLeft());
+    }
+  });
+  testCases.push(testCase);
+}
+
+describe("smoke test rendering of paths in urls.txt", () => {
+  for (const testCase of testCases) {
+    // eslint-disable-next-line no-loop-func
+    it(`it appropriately renders ${testCase.path}`, async () => {
+      const resp = await page.goto(`${BASE_URL}${testCase.path}`, { waitUntil: 'networkidle2' });
```

Comment:
`BASE_URL` is defined in the jest and puppeteer configuration I believe. We can just add the line `/* global BASE_URL */` at the top fo this file

Reply:
HAI! Sorry for the confusion, `BASE_URL` is defined in `puppeteer.config.js:4`. 

And since ESLint doesn't know about that, we'll need to whitelist that global variable in `.eslintrc:10`, like so:

```
globals:
  page: true
  browser: true
  context: true
  jestPuppeteer: true
  BASE_URL: true // <-- this
rules:
...
```

Then in theory we don't have to add `/* global BASE_URL */` for every file that uses `BASE_URL`

Thank you!